osp/admin.py

```diff
@@ -1,19 +1,53 @@
 from django.contrib import admin
 
 from osp.models import (
-    UserInformation, Form, Question, Choice,
-    Checkbox, Dropdown, Paragraph, ShortAnswer,
-    Date, Time, FileUpload, Answer, ChoiceValue,
-    CheckboxValue, DropdownValue, ParagraphValue, ShortAnswerValue,
-    DateValue, TimeValue, FileUploadValue, FormFeedback, ZulipStat
+    Answer,
+    Checkbox,
+    CheckboxValue,
+    Choice,
+    ChoiceValue,
+    Date,
+    DateValue,
+    Dropdown,
+    DropdownValue,
+    FileUpload,
+    FileUploadValue,
+    Form,
+    FormFeedback,
+    Paragraph,
+    ParagraphValue,
+    Question,
+    ShortAnswer,
+    ShortAnswerValue,
+    Time,
+    TimeValue,
+    UserInformation,
+    ZulipStat,
 )
 
 models = [
-    UserInformation, Form, Question, Choice,
-    Checkbox, Dropdown, Paragraph, ShortAnswer,
-    Date, Time, FileUpload, Answer, ChoiceValue,
-    CheckboxValue, DropdownValue, ParagraphValue, ShortAnswerValue,
-    DateValue, TimeValue, FileUploadValue, FormFeedback, ZulipStat
+    UserInformation,
+    Form,
+    Question,
+    Choice,
+    Checkbox,
+    Dropdown,
+    Paragraph,
+    ShortAnswer,
+    Date,
+    Time,
+    FileUpload,
+    Answer,
+    ChoiceValue,
+    CheckboxValue,
+    DropdownValue,
+    ParagraphValue,
+    ShortAnswerValue,
+    DateValue,
+    TimeValue,
+    FileUploadValue,
+    FormFeedback,
+    ZulipStat,
 ]
```

Comment:
You can do something like this
```
from django.apps import apps

models = apps.get_models()
```
**P.S.- Not tested**

Reply:
I think this import follows the code guidelines as many imports are in this format.